presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -269,6 +269,19 @@ and a file system location of ``s3://test_bucket/test_schema/test_table``:
         location = 's3://test_bucket/test_schema/test_table')
     )
 
+Session Properties
+-------------------
+
+Session properties set behaviour changes for queries executed within the given session.
+The following list is a partial list of the session properties available in the Iceberg connector.
+
+============================================= ===============================================================
+Property Name                                 Description
```

Comment:
Consider adding a Default column and a default value, similar to the [Configuration Properties](https://prestodb.io/docs/0.284/connector/iceberg.html#configuration-properties) in the Iceberg Connector doc and the [session property in the Hudi Connector](https://prestodb.io/docs/0.284/connector/hudi.html#configuration) doc.